.husky/pre-commit

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
```

Comment:
Looks like I can make it a node script instead

Reply:
Please

Reply:
want to try [zx](https://github.com/google/zx)? I haven't tried it yet.

Reply:
In order to change `#!/bin/sh` to `#!/usr/bin/env zx`, I think all devs would need to have zx globally installed. At least it didn't work for me with it only in `node_modules`. Dunno if that would be worth it?

Reply:
Turned off verbosity. Output now:

```
$ npx zx .shared/copy-shared-files.mjs 
Running pre-commit hook.
This will overwrite any changes made to config files in samples (like hello-world/tsconfig.json).
Proceed? [Y/n] y
Copying config files from .shared/ to samples... done.
```